example/test_problem/Hydro/CCSN/python_script/convert_progenitor_form.py

```diff
@@ -0,0 +1,164 @@
+#! /usr/bin/env python
+#
+# -*- coding: utf-8 -*-
+#
+#  Purpose:
+#    Converting the progenitor file format (WH07, Heager2005, Sukhbold2015, and MESA)
+#    to the format that GAMER can use for CCSN test problems
+#
+
+import argparse
+import sys
+import numpy as np
+
+
+def convert_WHS(file):
+
+    # read file info
+    f_info = file.readline()
+
+    # read the header
+    headers = []
+    line_header = file.readline()
+    headers = [word.strip() for word in line_header.strip().split("  ") if word]
+
+    for i in range(len(headers)):
+        keys = ["cell outer radius", "cell density", "cell temperature", "cell outer velocity",
+                "cell Y_e", "cell pressure", "cell angular velocity"]
```

Comment:
Why put this in the for loop? Same in `convert_MESA()`

Reply:
I removed the for loop.